src/WMStream.ts

```diff
@@ -0,0 +1,29 @@
+const util = require('util');
+const stream = require('stream');
+const Writable = stream.Writable;
+
+export default class WMStream extends Writable 
+{
```

Comment:
Please avoid using `any` whenever possible. We really should be enabling strict type checks. I understand you turned it off for prototyping, I've filed issues to enabled those TypseScript checks, meanwhile we need to make sure new code checked from this point on follows those rules.

Reply:
i agree, but for most of the ipfs packages, we dont have typescript definitions, hence cant add those types

Reply:
I agree for cases on IPFS packages, `any` maybe unavoidable in some cases unless we want to spend time writing definitions, however, in many cases such as in `BlobLock`, use of `any` can be completely avoided, even for callbacks, simpler types for `key` etc.